Date: Tue, Jun 28 2011 11:20:10
From: Seth Heasley <seth.heasley@intel.com>
References: bnc#682755 fate#311837
Subject: Core Driver Enabling for Intel Panther Point PCH
Patch-mainline: 2.6.39

Core Driver Enabling for the Intel Panther Point PCH.

This patch adds the DeviceIDs for HD Audio, SMBus, SATA, Watchdog, and LPC for the Intel Panther Point PCH.

Signed-off-by: Seth Heasley <seth.heasley@intel.com>
Acked-by: Jiri Kosina <jkosina@novell.com>

Automatically created from "patches.drivers/intel-add-partner-point-pch-support.patch" by xen-port-patches.py

--- sle11sp2-2011-07-15.orig/arch/x86/pci/irq-xen.c	2011-05-05 16:10:08.000000000 +0200
+++ sle11sp2-2011-07-15/arch/x86/pci/irq-xen.c	2011-07-15 14:02:36.000000000 +0200
@@ -606,8 +606,10 @@ static __init int intel_router_probe(str
 		return 1;
 	}
 
-	if ((device >= PCI_DEVICE_ID_INTEL_PCH_LPC_MIN) &&
-		(device <= PCI_DEVICE_ID_INTEL_PCH_LPC_MAX)) {
+	if ((device >= PCI_DEVICE_ID_INTEL_PCH_LPC_MIN &&
+		device <= PCI_DEVICE_ID_INTEL_PCH_LPC_MAX)
+	|| 	(device >=PCI_DEVICE_ID_INTEL_PANTHERPOINT_LPC_MIN &&
+		device <= PCI_DEVICE_ID_INTEL_PANTHERPOINT_LPC_MAX)) {
 		r->name = "PIIX/ICH";
 		r->get = pirq_piix_get;
 		r->set = pirq_piix_set;
